ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml

```diff
@@ -2194,23 +2204,21 @@ tests:
     env:
       E2E_RUN_TAGS: '@amd64 and @gcp-ipi and @proxy and @network-ovnkubernetes and
         not @fips'
-      FEATURE_SET: TechPreviewNoUpgrade
       TAG_VERSION: '@4.14'
     test:
     - chain: openshift-e2e-test-qe
-    workflow: cucushift-installer-rehearse-gcp-ipi-proxy
+    workflow: cucushift-installer-rehearse-gcp-ipi-proxy-techpreview
 - as: gcp-ipi-proxy-techpreview-p3-f28-destructive
   cron: 36 12 25 * *
   steps:
     cluster_profile: gcp-qe
     env:
       E2E_RUN_TAGS: '@amd64 and @gcp-ipi and @proxy and @network-ovnkubernetes and
         not @fips and @destructive'
-      FEATURE_SET: TechPreviewNoUpgrade
       TAG_VERSION: '@4.14'
     test:
     - chain: openshift-e2e-test-qe-destructive
-    workflow: cucushift-installer-rehearse-gcp-ipi-proxy
+    workflow: cucushift-installer-rehearse-gcp-ipi-proxy-techpreview
```

Comment:
@jinyunma just double check if cucushift-installer-rehearse-vsphere-ipi-proxy-techpreview-provision chain still need ref ipi-conf-techpreview?

per PR https://github.com/openshift/release/pull/38872, it is not needed in TP workflow.

if it is not needed after confirmation, please remove it from other TP workflow chain, like cucushift-installer-rehearse-aws-ipi-proxy-techpreview-po-provision

Reply:
For 4.12+, set FEATURE_SET on job config level as what JoelSpeed did since it is supported to be configured in install-config.yaml starting from 4.12. 
For old version ,still need ref ipi-conf-techpreview to enabel TP.